protocols/1028-mousenor-inc/odorant_screening.ot2.py

```diff
@@ -1,93 +1,166 @@
     p50.transfer(
         30,
         dilution_plate.wells('A1', length=6),
         dilution_plate.wells('B1', length=6),
-        mix_after=(3, 15))
+        mix_after=(10, 50))
+
+    # set pipette flow rate back to default
+    m300.set_flow_rate(aspirate=150, dispense=300)
+    p50.set_flow_rate(aspirate=25, dispense=50)
 
     # transfer standards to ELISA plate
-    m300.transfer(
-        60,
-        dilution_plate.cols('1'),
-        [col for col in elisa_plate.cols('1', to='2')])
+    m300.pick_up_tip()
+    for col in standard_dest:
+        m300.transfer(
+            60, dilution_plate.cols('1'), col[0].top(), new_tip='never')
+    m300.drop_tip()
+    update_multi_tip_count(1)
 
     # transfer samples to ELISA plate
-    sample_source = sample_plate.cols(sample_source_col, length=sample_cols)
-    sample_dest = elisa_plate.cols(sample_dest_col, length=sample_cols)
-    m300.transfer(60, sample_source, sample_dest, new_tip='always')
+    for source, dest in zip(samples, sample_dest):
+        m300.transfer(60, source, dest)
+        update_multi_tip_count(1)
 
-    plate_loc = elisa_plate.cols('1', to='2')+sample_dest
+    plate_loc = standard_dest + sample_dest
 
     # transfer dilution cAMP-AP to plate
-    m300.pick_up_tip()
     for col in plate_loc:
-        m300.transfer(30, diluted_cAMP, col.top(), new_tip='never')
-    m300.drop_tip()
+        m300.transfer(30, diluted_cAMP, col, mix_after=(2, 30))
+        update_multi_tip_count(1)
 
     # transfer antibody to plate and mix
     for col in plate_loc:
-        m300.transfer(60, antibody, col, mix_after=(3, 30))
+        m300.transfer(60, antibody, col, mix_after=(2, 30))
+        update_multi_tip_count(1)
 
     m300.delay(minutes=60)
 
     # remove solution from wells
-    m300.transfer(150, plate_loc, liquid_trash, new_tip='always')
+    for col in plate_loc:
+        m300.transfer(150, col, liquid_trash)
+        update_multi_tip_count(1)
 
     # wash 6x with wash buffer
+    m300.pick_up_tip()
+    start_tip = m300.current_tip()
     for cycle in range(6):
-        for col in plate_loc:
+        if not m300.tip_attached:
+            m300.start_at_tip(start_tip)
             m300.pick_up_tip()
-            m300.transfer(200, wash_buffer, col, new_tip='never')
-            m300.transfer(200, col, liquid_trash, new_tip='never')
-            m300.drop_tip()
+        m300.distribute(
+            200, wash_buffer, [col[0] for col in plate_loc], trash=False)
+        m300.transfer(200, plate_loc, liquid_trash, trash=False)
+    update_multi_tip_count(2)
 
     # add substrate/enhancer solution to plate
     m300.pick_up_tip()
     for col in plate_loc:
-        m300.transfer(100, substrate_solution, col.top(), new_tip='never')
+        m300.transfer(100, substrate_solution, col[0].top(), new_tip='never')
     m300.drop_tip()
+    update_multi_tip_count(1)
+
+    # incubate at room temperature for 30 minutes
+    m300.delay(minutes=30)
```

Comment:
I think having a delay at the end of a protocol file might cause smoothie issues. Can you remove this?